Chaincase.UI/wwwroot/style/global.css

```diff
@@ -1,3 +1,57 @@
+﻿/** Global **/
+* {
+    box-sizing: border-box;
+}
+
+.danger-text {
+    font-weight: bold;
+    color: darkred;
+    font-size: 16px;
+}
+
+/** CoinJoin Page **/
+
+.list-group {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+}
+
+.list-group li {
+    border-bottom: 1px solid #ddd;
+    margin-top: -1px; /* Prevent double borders */
+    padding: 10px;
+    font-size: 13px;
```

Comment:
Y U NO REM?

https://css-tricks.com/rems-ems/ final method might be the move

Reply:
Yeah, that's fair. But I think I'm also just going to remove this CSS in favor of `ion-list`